### PR TITLE
feat: ✨  list, `in` operator and `size` function support

### PR DESCRIPTION
**Pull request by alvaro-saavedra-sumup**
## Describe your changes

Add support for lists and list operations, used spects from cel-spec.

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests (if possible)

### CODE REVIEW

File: src/spec/lists.spec.ts [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+    it('should return correct list if right side is empty', () => {
+      const expr = '[1, 2] + []'
+
+      const result = evaluate(expr)
+
+      expect(result).toStrictEqual([1, 2])
+    })
+
+    it.todo('should throw an error if lists have different types', () => {
```
Comment (ChromeGG) [inline src/spec/lists.spec.ts:123]: I've checked it [against playground](https://playcel.undistro.io/).

**Review by ChromeGG — COMMENTED** [review]
Amazing work! Thank you for contribution! <3

There are some small thing to adjust, I left todos. I can start fixing it in 10h

**Comment by ChromeGG** [comment]
Note to myself: do NOT forget to update `demo.ts` and release it :)

**Comment by ChromeGG** [comment]
Unfortunately there is a couple of todos we should address. If you really need this feature released soon, I can marge and release it in this form. Otherwise I prefer to rethink the implementation to handle edge-cases like `[1,2,3][1]` and `[1][5]`

**Comment by alvaro-saavedra-sumup** [comment]
> Unfortunately there is a couple of todos we should address. If you really need this feature released soon, I can marge and release it in this form. Otherwise I prefer to rethink the implementation to handle edge-cases like `[1,2,3][1]` and `[1][5]`

Don't worry, let me check how to address them. I think I will work on them this week end.

File: src/spec/lists.spec.ts (hunk 1 above)
Comment (alvaro-saavedra-sumup) [reply src/spec/lists.spec.ts]: Implemented the check

**Review by ChromeGG — APPROVED** [review]

**Pull request merged**